pkg/operator/baremetal_config.go

```diff
@@ -65,3 +77,89 @@ func getBaremetalProvisioningConfig(dc dynamic.Interface, configName string) (Ba
 		ProvisioningDHCPRange:    provisioningDHCPRange,
 	}, nil
 }
+
+func getProvisioningIPCIDR(baremetalConfig BaremetalProvisioningConfig) string {
+	if baremetalConfig.ProvisioningIp != "" {
+		cidr := strings.Split(baremetalConfig.ProvisioningNetworkCIDR, "/")
+		return baremetalConfig.ProvisioningIp + "/" + cidr[1]
+	} else {
+		return unknownProvisioningConfig
+	}
+}
+
+func getDeployKernelUrl(baremetalConfig BaremetalProvisioningConfig) string {
+	if baremetalConfig.ProvisioningIp != "" {
+		return baremetalConfig.ProvisioningIp + ":" + baremetalHttpPort + baremetalKernelUrlSubPath
+	} else {
+		return unknownProvisioningConfig
+	}
+}
+
+func getDeployRamdiskUrl(baremetalConfig BaremetalProvisioningConfig) string {
+	if baremetalConfig.ProvisioningIp != "" {
+		return baremetalConfig.ProvisioningIp + ":" + baremetalHttpPort + baremetalRamdiskUrlSubPath
```

Comment:
Ditto, URL should be constructed like above

Reply:
Done.